src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -57,6 +58,10 @@ protected static ConfigDef baseConfigDef() {
           .define(MAX_IN_FLIGHT_REQUESTS_CONFIG, Type.INT, 5, Importance.MEDIUM,
                   "The maximum number of indexing requests that can be in-flight to Elasticsearch before blocking further requests.",
                   group, 5, Width.SHORT, "Max In-flight Requests")
+          .define(NODE_DISCOVERY_FREQUENCY, Type.LONG, 60, Importance.MEDIUM,
```

Comment:
I agree that this feature is pretty important especially when using Elasticsearch in the cloud.